invenio_cli/commands/services.py

```diff
@@ -296,6 +302,7 @@ def setup(self, force, demo_data=True, stop=False, services=True):
 
         steps.extend(self._setup())
         steps.extend(self.fixtures())
+        steps.extend(self.declare_queues())
```

Comment:
As Max pointed out `invenio-queues` only comes after v12. Therefore it needs an `rdm_version >= 12` if-condition. I am wondering if this could be moved inside `_setup` (since it should run before fixtures in case we for example want demo stats).  Then it could be moved below this `if`:

https://github.com/inveniosoftware/invenio-cli/blob/f165c58ceafa15c1518ebbe528039432b1570afe/invenio_cli/commands/services.py#L219

Reply:
Very good point! 👍🏼